<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simentryForm</class>
 <widget class="QWidget" name="simentryForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_simEntry">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Simulator Log Entry</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>127</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="sim_dateLabel">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QDateEdit" name="sim_dateEdit">
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
       <property name="date">
        <date>
         <year>2001</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="sim_authorLabel">
       <property name="text">
        <string>Auth N#</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="sim_authorCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sim_timeLabel">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="sim_funcLabel">
       <property name="text">
        <string>Function</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="sim_funcCombo">
       <item>
        <property name="text">
         <string>TRAINEE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SFI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SFE</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="remarks_label">
       <property name="text">
        <string>Remarks</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QTextEdit" name="sim_remarksText"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QTimeEdit" name="sim_timeEdit">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
       </property>
       <property name="minimumTime">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="displayFormat">
        <string>hh:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
       <property name="time">
        <time>
         <hour>22</hour>
         <minute>59</minute>
         <second>59</second>
        </time>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_new_2">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/adept_install.png</normaloff>:/images/images/adept_install.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete_2">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/adept_purge.png</normaloff>:/images/images/adept_purge.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_revert_2">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/button_cancel.png</normaloff>:/images/images/button_cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_submit_2">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/images/images/button_ok.png</normaloff>:/images/images/button_ok.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_simLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>731</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Simulator Log</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="first_pb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>31</horstretch>
          <verstretch>31</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/images/go-first.png</normaloff>:/images/images/go-first.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="prev_pb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>31</horstretch>
          <verstretch>31</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/images/go-previous.png</normaloff>:/images/images/go-previous.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="book_combo"/>
      </item>
      <item>
       <widget class="QPushButton" name="next_pb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>31</horstretch>
          <verstretch>31</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/images/go-next.png</normaloff>:/images/images/go-next.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="last_pb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>31</horstretch>
          <verstretch>31</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/images/go-last.png</normaloff>:/images/images/go-last.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="sim_logView"/>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>220</y>
     <width>731</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>421</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Simulators</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLineEdit" name="auth_lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="type_label">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="type_lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="level_label">
         <property name="text">
          <string>Level</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="level_lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="auth_label">
         <property name="text">
          <string>Auth N#</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_new">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/images/adept_install.png</normaloff>:/images/images/adept_install.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_delete">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/images/adept_purge.png</normaloff>:/images/images/adept_purge.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_submit">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/images/button_ok.png</normaloff>:/images/images/button_ok.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_revert">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/images/button_cancel.png</normaloff>:/images/images/button_cancel.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QTableView" name="simulatorsTable"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
